### PR TITLE
Editorsection

### PR DESCRIPTION
**Pull request by andypasso**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+}
+
+.first {
+    margin-bottom: 20%;
+}
+
+
+#g-ai1 {
+    float: left;
+    margin-right: 9%;
```
Comment (mosaaleb) [inline style.css]: `g-ai1` through `g-ai4` have the same style properties, we can remove alot of duplicate code with a class having the same properties, and give custom style properties to those who need a little change. Also `g-ai5` through `g-ai8`.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+}
+
+.article-version a {
+    text-decoration: none;
+    color: #999999;
+    font-size: 12px;
+}
+
+.commentsandsocial {
+    display: flex;
```
Comment (mosaaleb) [inline style.css]: The default value of `flex-direction` is row, so we don't actually need this line.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+
+.label-image-editorsection time {
+    font-size: 0.75rem;
+    text-align: left;
+}
+
+.bottomsection {
+    display: flex;
+    justify-content: flex-start;
+    width: 100%;
```
Comment (mosaaleb) [inline style.css]: Also, the default value of `justify-content` is `flex-start`

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Amazing work @andynarf, @AndresFMoya :clap:

#### The UI is neat and clean. Kudos on using separate `normalize.css` file. Your code is organized and easy to read. Still some requirements needs to be handled first before merging.

##### Navigation bar:
- [x] Uses the `<nav>` tag for more semantic html
- [x] Has the `hamburger` and `search` icons in the leftmost side, you can use font awesome.

##### Footer
- [x] The NYTimes logo is a link.


##### General:
- [x] Let's fill the relevant information the `readme.md` file.
- [x] Let's make sure that we follow the style guidelines by adding [stickler](https://github.com/microverseinc/linters-config/tree/master/css), and then make sure that all checks have passed.

Please submit a new code review once you handle these requirements. 

Keep up the good work team. Happy coding :computer:

File: style.css [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+.article-version a {
+    text-decoration: none;
+    color: #999999;
+    font-size: 12px;
+}
+
+.commentsandsocial {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```
Comment (mosaaleb) [inline style.css]: The default value of `flex-direction` is row, so we don't actually need this line.

**Comment by andypasso** [comment]
we had a lot of failures with our normalize.css on stickler. 
I don't know they are actual issues.

**Comment by mosaaleb** [comment]
All of them had to do with style guidelines. You can check the errors in the checks tab on top of this pull request page. Also since this file is not written by you you can just [ignore](https://stackoverflow.com/questions/42070748/vs-code-style-lint-ignore-directories) it

**Comment by andynarf** [comment]
yeah, i know , ok, I'll just use a different css reset file, since I see no point in correcting every single line on that one.

**Comment by andypasso** [comment]
there, we fixed everything =D

**Comment by samgaco** [comment]
Hi, @andynarf  and @mosaaleb :vulcan_salute: 

Good work! Still, some things left to add;

* You should implement the subscribe section in the footer and the bottom part of the footer should include links with copyright, contact us, help, etc

left

* Any section including paragraphs in your project should be enclosed within the` <section> `tag. In the case of the "Editor's picks" and the "Sign up" button they should also include the `<aside>` tag. Learn more about it here: https://www.w3schools.com/html/html5_semantic_elements.asp

* Correct the errors shown in https://validator.w3.org/ for the HTML file

Commit the changes and submit back for a code review when you are done. Happy Coding! :muscle:

**Comment by bolah2009** [comment]
### Hey Team

Good job :clap: 
The previous [review](https://github.com/andynarf/Positioning_and_Floating_Elements/pull/1#issuecomment-513372753) pointed out some changes that are yet to be made.
Check out this [comment](https://github.com/andynarf/Positioning_and_Floating_Elements/pull/1#pullrequestreview-265222286) for more details

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Nice Job Team! :clap: 

You've done an awesome Job :muscle:

- A few changes to be made, please check the screenshot below and kindly make changes accordingly. Check the [original site](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) for more details.

| Current | Preffered |
| :---:   | :---:   |
| syb btn current  | sub login preffered  |
| inflation current  |  pos-n-flo |
| read more btn current  | read button preffered  |
| NOT FOUND  | editor's pick  |
| NOT FOUND  | sign up  |
| m popular sec nty current |  nyt popular preffered |
| editor pick sec nyt | nyt editor preffered  |
| subcribe sec nyt current  | nyt footer preffered  |

You have done an awesome job, consider making these changes, taking into consideration the **positioning and styling** of the above-screenshotted components of the page.

Good luck with your next submission :+1: 

Cheers
Bola

**Review by bolah2009 — APPROVED** [review]
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

### The [page](https://raw.githack.com/andynarf/Positioning_and_Floating_Elements/editorsection/index.html) looks good to me, however check out the code with [HTML Validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fandynarf%2FPositioning_and_Floating_Elements%2Feditorsection%2Findex.html) and on 100% browser zoom.


Good luck with your next project
Keep it up :star2:

Cheers
Bola

**Pull request merged**